lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -69,23 +69,13 @@ public function prepare($prepareString, $driverOptions = [])
      */
     public function query()
```

Comment:
What happens if we declare a variadic argument here, btw? Would that break inheritance in any way?

Reply:
Yes:
> Fatal error: Declaration of Doctrine\DBAL\Driver\MyConnection::query() must be compatible with Doctrine\DBAL\Driver\PDOConnection::query(...$args)